README.md

```diff
@@ -1,81 +1,46 @@
-  </a>
-    <img alt="Sims" src="https://github.com/cosmos/cosmos-sdk/workflows/Sims/badge.svg" />
-    <img alt="Lint Satus" src="https://github.com/cosmos/cosmos-sdk/workflows/Lint/badge.svg" />
-</div>
+Larger modifications include:
 
-The Cosmos SDK is a framework for building blockchain applications. [Tendermint Core (BFT Consensus)](https://github.com/tendermint/tendermint) and the Cosmos SDK are written in the Golang programming language. Cosmos SDK is used to build [Gaia](https://github.com/cosmos/gaia), the first implementation of the Cosmos Hub.
+1. Early adoption of `PrepareProposal` and `ProcessProposal`. This was added to the fork because at the time of development, a Cosmos SDK release was not available with these ABCI methods. Ref: https://github.com/celestiaorg/cosmos-sdk/commit/233a229cabf0599aed91b6b6697c268753731b2c
+1. The addition of `chainID` to baseapp so that a branch of state can be used in PrepareProposal and ProcessProposal. Ref: https://github.com/celestiaorg/cosmos-sdk/pull/326
+1. The consensus params version is overriden to the `AppVersion` to enable EndBlocker to update the `AppVersion`. Ref: https://github.com/celestiaorg/cosmos-sdk/pull/321
 
-**WARNING**: The Cosmos SDK has mostly stabilized, but we are still making some
-breaking changes.
+Smaller modifications include:
 
-**Note**: Requires [Go 1.19+](https://go.dev/dl)
+1. The addition of a `SetTxDecoder` on tx config so that celestia-app can override the default tx decoder with one that supports decoding `BlobTx`s. Ref: https://github.com/celestiaorg/cosmos-sdk/pull/311
+1. The addition of a `start_time` to the vesting module's `MsgCreateVestingAccount` so that vesting accounts can be created with a delayed start time. Ref: https://github.com/celestiaorg/cosmos-sdk/pull/342
+1. Allow celestia-app to override the default consensus params via the `init` command. Ref: https://github.com/celestiaorg/cosmos-sdk/pull/317
 
-## Quick Start
+Modifications that make it easier to maintain this fork:
 
-To learn how the Cosmos SDK works from a high-level perspective, see the Cosmos SDK [High-Level Intro](./docs/intro/overview.md).
+1. Modify CODEOWNERS to Celestia maintainers
+1. Modify Github CI workflows to include `release/**` branches
+1. Modify Github CI workflows to not run some workflows
+1. Delete cosmovisor
 
-If you want to get started quickly and learn how to build on top of Cosmos SDK, visit [Cosmos SDK Tutorials](https://tutorials.cosmos.network). You can also fork the tutorial's repository to get started building your own Cosmos SDK application.
+Modifications that may be revertable:
 
-For more information, see the [Cosmos SDK Documentation](./docs/).
+1. Override the default keyringBackend from `os` to `test`. Maybe move to celestia-app
+1. Increase `DefaultGasLimit` from 200000 to 210000.
+1. Remove `Evidence` from grpc/tmservice/types.pb.go.
+1. The addition of `SignLedgerAminoJSON` in keyring b/c Ledger issue.
```

Comment:
this line can be removed if we merge https://github.com/celestiaorg/cosmos-sdk/pull/357

```suggestion
```